regression/contracts/loop_contracts_do_while/test.desc

```diff
@@ -7,3 +7,4 @@ main.c
 --
 --
 This test checks that loop contracts work correctly on do/while loops.
+Fails because contracts are not yet supported on do while loops.
```

Comment:
This one should pass with loop invariant `0<= x && x < 10`

Reply:
I don't think so: this one is `contracts`, not `contracts-dfcc` -- but what you are saying is true for `regression/contracts-dfcc/loop_contracts_do_while/test.desc`, will fix.

Reply:
my bad.